db/silkworm/db/bitmap.hpp

```diff
@@ -30,6 +30,9 @@ roaring::Roaring64Map read(ByteView serialized);
 // See TG SeekInBitmap64.
 std::optional<uint64_t> seek(const roaring::Roaring64Map &bitmap, uint64_t n);
 
+// Return cut bitmap of given size limit
+roaring::Roaring64Map cut_left(roaring::Roaring64Map *bitmap, uint64_t len);
```

Comment:
If input bitmap doesn't change, then it should be marked as `const`. Also, it may not be `nullptr`, so it should be a reference rather than pointer. See [Google Style on Inputs and Outputs](https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs).
In other words, the function signature should probably be
```
roaring::Roaring64Map cut_left(const roaring::Roaring64Map& bitmap, uint64_t len);
```

Reply:
it changes

Reply:
OK, then the function doc should say how it changes.
A.t.m it's not clear from the doc what happens to the input and what is returned.
Also, it should be passed by reference, not by pointer, to highlight that it can't be `nullptr`.